### PR TITLE
Added week 2 blog

### PR DESCRIPTION
**Pull request by xtanion**
Docs are generating fine and it's visible in blogs.html
image

### CODE REVIEW

File: docs/source/posts/2022/2022-06-29-week2_blog.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+This week I worked primarily on the glTF fetcher and exporting scenes to a glTF file. I added tests and docstrings for all functions. I modified the ``fetch_gltf`` function to return the downloaded files. As we planned, the PR for the glTF fetcher was merged this week.
+
+I fixed the color issue of the glTF exporter by manually appending the color data from the actor to the ``polydata``. But there's another issue raised while using actors from ``vtkSource``. The ``utils.get_polydata_triangles(polydata)`` method only works with primitives and it raises error when used with ``vtkSource`` actors.
+
+Textures and Cameras can now be added to the glTF file. However, it only supports base texture only, I'll be working on materials support later on.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-06-29-week2_blog.rst:16]: Last week as you said, it should be `baseTexture`

File: docs/source/posts/2022/2022-06-29-week2_blog.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+What is coming up next week?
+----------------------------
+
+* Saving all models download link (from the Khronos gltf-samples repository) to a JSON file, create a separate branch and add the download script.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-06-29-week2_blog.rst:22]: Maybe `glTF`

File: docs/source/posts/2022/2022-06-29-week2_blog.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* Add tests and docstring for PR `#600 <https://github.com/fury-gl/fury/pull/600>`_.
+* Create a PR for glTF exporter.
+
+Did you get stuck anywhere?
+---------------------------
+
+* I managed to fix colors in polydata by adding them manually but It raised another issue with indices (triangles) of the actor weren't of the correct shape. We decided to take a look at it later.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-06-29-week2_blog.rst:29]: comma before `but`

Comment (ganimtron-10) [inline docs/source/posts/2022/2022-06-29-week2_blog.rst:29]: lower case `i` in `but It`

**Review by ganimtron-10 — COMMENTED** [review]
Hi @xtanion ,
The blog was generated successfully.
I have mentioned a few suggestions below. PTAL.
Thanks!

File: docs/source/posts/2022/2022-06-29-week2_blog.rst [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+This week I worked primarily on the glTF fetcher and exporting scenes to a glTF file. I added tests and docstrings for all functions. I modified the ``fetch_gltf`` function to return the downloaded files. As we planned, the PR for the glTF fetcher was merged this week.
+
+I fixed the color issue of the glTF exporter by manually appending the color data from the actor to the ``polydata``. But there's another issue raised while using actors from ``vtkSource``. The ``utils.get_polydata_triangles(polydata)`` method only works with primitives and it raises error when used with ``vtkSource`` actors.
+
+Textures and Cameras can now be added to the glTF file. However, it only supports baseTexture only, I'll be working on materials support later on.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-06-29-week2_blog.rst:16]: The comma after `only` should be a full stop

**Review by m-agour — COMMENTED** [review]
Hello @xtanion
There are no typos but some punctuation issues that I'll mention below.

File: docs/source/posts/2022/2022-06-29-week2_blog.rst (hunk 4 above)
Comment (m-agour) [inline docs/source/posts/2022/2022-06-29-week2_blog.rst:16]: The comma after `baseTexture only` should be a full stop

File: docs/source/posts/2022/2022-06-29-week2_blog.rst [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+   :author: Shivam Anand
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+This week I worked primarily on the glTF fetcher and exporting scenes to a glTF file. I added tests and docstrings for all functions. I modified the ``fetch_gltf`` function to return the downloaded files. As we planned, the PR for the glTF fetcher was merged this week.
+
+I fixed the color issue of the glTF exporter by manually appending the color data from the actor to the ``polydata``. But there's another issue raised while using actors from ``vtkSource``. The ``utils.get_polydata_triangles(polydata)`` method only works with primitives and it raises error when used with ``vtkSource`` actors.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-06-29-week2_blog.rst:14]: Should be `raises an error`

**Comment by skoudoro** [comment]
Hi @xtanion,

After you addressed @m-agour  comments, it should be good to go. 

Thanks.

**Comment by skoudoro** [comment]
Thanks @xtanion, merging

**Pull request merged**